appengine/taskqueue/pull-counter/pullcounter_test.py

```diff
@@ -31,3 +32,11 @@ def test_app(testbed):
     tasks = tq_stub.get_filtered_tasks()
     assert len(tasks) == 1
     assert tasks[0].name == 'task1'
+
+    with mock.patch('main.update_counter') as mock_update:
+        # Force update to fail, otherwise the loop will go forever.
```

Comment:
I'm probably missing some context, but is it not weird to have a worker that polls indefinitely? I'd expect a cron job or something instead, which you can at least cancel?